### PR TITLE
Minimap icons change when marines die

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Different icons for when you are dead or even undefibbable.

## Why It's Good For The Game

More info on minimap

## Changelog
:cl:
expansion: Minimap icons change when marines die
/:cl:

### CODE REVIEW

**Comment by Wayland-Smithy** [comment]
How hard would it be to also change miner icons if they are at full integrity?

**Comment by BraveMole** [comment]
> How hard would it be to also change miner icons if they are at full integrity?

real easy

**Comment by joniroq** [comment]
How hard would it be to add a "You are Here" icon to the minimap?

**Comment by BraveMole** [comment]
> How hard would it be to add a "You are Here" icon to the minimap?

i've looked into, harder

**Comment by XSlayer300** [comment]
I think there's one problem. It kinda defeats the purpose of seeing who is dead and defiabble on the MedHUD, right?

**Comment by BraveMole** [comment]
> I think there's one problem. It kinda defeats the purpose of seeing who is dead and defiabble on the MedHUD, right?

well, don't think it's really an issue. You can just observe the body anyway to know that

File: code/modules/mob/death.dm [hunk 1]
```diff
@@ -39,15 +39,15 @@
...
+	set_stat(DEAD)
+
 	SEND_GLOBAL_SIGNAL(COMSIG_GLOB_MOB_DEATH, src)
 	SEND_SIGNAL(src, COMSIG_MOB_DEATH, gibbing)
 	log_combat(src, src, "[deathmessage]")
 
 	if(deathmessage && !silent && !gibbing)
 		visible_message("<b>\The [name]</b> [deathmessage]")
 
-	set_stat(DEAD)
```
Comment (TiviPlus) [inline code/modules/mob/death.dm:49]: Bad

File: code/__DEFINES/dcs/signals.dm [hunk 2]
```diff
@@ -358,6 +358,8 @@
 ///from [/mob/living/carbon/human/proc/remove_overlay]: (cache_index, list/overlays_to_remove)
 #define COMSIG_HUMAN_REMOVE_OVERLAY "human_overlay_removed"
 
+#define COMSIG_HUMAN_UNDEFIBBABLE "human_set_undefibbable"
+
 // shuttle signals
```
Comment (TiviPlus) [inline code/__DEFINES/dcs/signals.dm:361]: Human set undefibbable

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**